.github/workflows/sonar.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Set up JDK 11
```

Comment:
Looks like you updated the java-version. If this works, can you also update the name of this step @9034725985

Reply:
Good catch, @patstha I will update now.

Looks like sonar is good now. 
![image](https://github.com/patstha/mydotnet/assets/1676445/93019663-c369-40ab-b237-024843e5ae4e)